bindings/python/triplestore/backends/ontopy.py

```diff
@@ -0,0 +1,197 @@
+import io
+import os
+import tempfile
+import warnings
+from typing import TYPE_CHECKING
+
+from ontopy.ontology import get_ontology, Ontology, _unabbreviate
+
+
+from dlite.triplestore import Literal
+
+if TYPE_CHECKING:  # pragma: no cover
+    from collections.abc import Sequence
+    from typing import Generator
+
+    from dlite.triplestore import Triple
+
+
+class OntopyStrategy:
+    """Triplestore strategy for EMMOntoPy.
+
+    Arguments:
+        onto: Ontology to initiate with.  Defaults to an new ontology with
+            `base_iri`.
```

Comment:
```suggestion
        base_iri: The base iri of the ontology.
        onto: Ontology to initiate with.  Defaults to an new ontology with
            `base_iri`.
```

Reply:
Thanks for the correction. I also improved the documentation of `onto`